apps/frontend/Dockerfile

```
@@ -0,0 +1,41 @@
+FROM node:16-alpine AS base
+
+FROM base AS builder
+RUN apk add --no-cache libc6-compat
+RUN apk update
+# Set working directory
+WORKDIR /app
+RUN yarn global add turbo
+COPY . .
+RUN turbo prune --scope=@dundring/frontend --docker
+
+# Add lockfile and package.json's of isolated subworkspace
+FROM base AS installer
+RUN apk add --no-cache libc6-compat
+RUN apk add --no-cache openssl1.1-compat
+RUN apk update
+WORKDIR /app
+
+# First install dependencies (as they change less often)
+COPY .gitignore .gitignore
+COPY --from=builder /app/out/json/ .
+COPY --from=builder /app/out/yarn.lock ./yarn.lock
+RUN yarn install --ignore-scripts
```

```suggestion
RUN yarn install --frozen-lockfile --ignore-scripts
```
Or else the lockfile isn't used, as i understand?

https://classic.yarnpkg.com/en/docs/cli/install#toc-yarn-install-frozen-lockfile